### ISSUE TITLE
Pink Promise in 8 Clues

### ISSUE THREAD

**Issue by iamwhoiamhahaha** [hanabi/hanabi.github.io#680]
newduke — Today at 10:19 AM
lvl max. What is the focus of a (t1) 5 stall? https://hanab.live/shared-replay/580059#2

IAMJEFF — Today at 10:21 AM
no focus

newduke — Today at 10:23 AM
what is the pink promise

IAMJEFF — Today at 10:23 AM
slot 4 iirc

newduke — Today at 10:23 AM
that's my thought

IAMJEFF — Today at 10:23 AM
but probably no focus in 2p turn 1

newduke — Today at 10:23 AM
5's always rightmost
so no pp?

IAMJEFF — Today at 10:24 AM
i would mark nothing from the turn 1 stall

newduke — Today at 10:24 AM
i see
how about t2?

IAMJEFF — Today at 10:24 AM
slot 4
as a 5cm

newduke — Today at 10:26 AM
ah
so only because cm?

IAMJEFF — Today at 10:26 AM
you mean the reason why the focus is on slot 4?

newduke — Today at 10:27 AM
yes
what if a stall were legal...
like dda or whatev

IAMJEFF — Today at 10:27 AM
probably slot 1 imo
so only because cm?
yes

newduke — Today at 10:28 AM
kk ty

IAMJEFF — Today at 10:44 AM
probably slot 1 imo
the doc says/defines slot 4 but i think it just depends on the situation. to minimize complex, the focus should be on rightmost for sure in favor of pp with a 5cm

EGG-SPLOSION!!!! — Today at 12:55 PM
https://hanab.live/shared-replay/580161#7
Variant: Omni-Ones & Omni
Convention Level: Max
he has context that his o4 is not a 2 save because Fey discarded earlier

newduke — Today at 1:00 PM
This example does not demonstrate what I am referring to by "bubblegum cannot be in reverse." In this example, the 2 is a save which cannot be a 2 and works as a bubblegum.  In another game, let Alice clue Bob a 3 to get cathy's i1 in slot 3. This will not work if there is a playable 3 because Bob does not have to wait for a bubblegum in reverse
this is the same as 5 pull finesse. Not allowed in reverse for similar logic

EGG-SPLOSION!!!! — Today at 1:06 PM
Yea, can we have someone clarify? I assume my clue would work in all cases, not just with the context that it is not a 2 that this clue works.

newduke — Today at 1:18 PM
Another omni question: can you break pink promise to fake save? https://hanab.live/shared-replay/580170#17
same game: can duke lie about a chop omni five to get b2 to tempo> https://hanab.live/shared-replay/580170#19
(could he be cluing the o5 or o1 a 2)

EGG-SPLOSION!!!! — Today at 1:19 PM
I was explaining to newduke that on Turn 17 you're allowed to break prink promise if slot 3 is also a critical card. For example if it's o5 on chop and b4 on slot 3....he can't just clue 5 to "promise" as I will then discard b4.   BTW newduke was 8 clue locked when giving the 4 clue.
New duke was saying you should always promise, and that he should always bomb his chop to double save me if it isn't o4 on chop.
On Turn 19, if Kakashi has o5 on slot 1, he can't clue blue and cluing 5 to 5 save is bad because then kakashi will be locked and burn a clue. Thus, it's better to just clue 2 to tempo the b2 in addition to picking up o5
Basically, what I'm trying to say is...there are situations where you're allowed to break pink promise and these two situations were those cases.  Can someone else weigh in and clarify?

**Comment by iamwhoiamhahaha** [comment]
imo, pp in 8 clues doesn't apply to 5 clues or any clue that fake saves

**Comment by iamwhoiamhahaha** [comment]
Close it since it looks like an issue for 2p
https://github.com/iamwhoiamhahaha/hanabi/issues/15
